entities/params/auth.go

```diff
@@ -0,0 +1,21 @@
+package params
+
+// PostAuthenticate represents request body for POST /api/authenticate
+type PostAuthenticate struct {
+	Username string `form:"username" validate:"required,email"`
+	Password string `form:"password" validate:"required"`
+}
+
+func (p *PostAuthenticate) TrimSpaces() {
+}
+
+// PostSignUp represents request body for POST /api/signup
+type PostSignUp struct {
+	Email         string `form:"email" validate:"required,email"`
+	Password      string `form:"password" validate:"required,min=8,max=191"`
```

Comment:
@Gligor23 we don't limit passwords, remove the max=191 validation from all Password fields.